make_modules.py

```
@@ -0,0 +1,23 @@
+import re
+
+with open('/tmp/telemetry.rs', 'r') as f:
+    lines = f.readlines()
+
+def find_block(start_line_prefix, include_impls=True, stop_before=None):
```

Ruff (0.15.9)

[warning] Boolean default positional argument in function definition

(FBT002)